hazelcast/src/main/java/com/hazelcast/sql/SqlColumnMetadata.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.sql;
+
+import javax.annotation.Nonnull;
+
+/**
+ * SQL column metadata.
+ */
+public final class SqlColumnMetadata {
+
+    private final String name;
+    private final SqlColumnType type;
+
+    @SuppressWarnings("ConstantConditions")
+    public SqlColumnMetadata(@Nonnull String name, @Nonnull SqlColumnType type) {
```

Comment:
What about marking this constructor a `@PrivateApi`? It's not supposed to be used by the users, we might add more metadata to it.
Also `SqlRowMetadata`, maybe also `SqlException`.

Reply:
Agreed, fixed.